great_expectations_cloud/agent/actions/draft_datasource_config_action.py

```diff
@@ -75,10 +75,10 @@ def _get_table_names(self, datasource: Datasource) -> list[str]:
 
     def _update_table_names_list(self, config_id: UUID, table_names: list[str]) -> None:
         with create_session(access_token=self._auth_key) as session:
-            response = session.patch(
-                url=f"{self._base_url}/organizations/"
-                f"{self._organization_id}/datasources/drafts/{config_id}",
-                json={"table_names": table_names},
+            url = f"{self._base_url}/api/v1/organizations/{self._organization_id}/draft-table-names/{config_id}"
```

Comment:
Which v1 endpoint should this be hitting? The fn name suggests `draft-table-names` but the previous impl suggests `draft-datasources`

Current impl works for integration, but fails unit

Reply:
tyty